app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -68,12 +84,44 @@ class AuthRepositoryTest : ShouldSpec(
 			verify { firebaseAuth.currentUser!!.delete() }
 		}
 
+		should("call bitmapConverter.convertBitmapToBytes WHEN updateUserAvatar called") {
+			every { firebaseAuth.currentUser!!.uid } returns userUid
+			every { storageReference.putBytes(any()) } returns mockk {
+				every { isComplete } returns true
+				every { exception } returns null
+				every { isCanceled } returns false
+				every { result } returns mockk()
+			}
+
+			tested.updateUserAvatar(userPhoto)
+
+			verify { bitmapConverter.convertBitmapToBytes(userPhoto, quality) }
+		}
+
+		should("call StorageReference.putBytes WHEN updateUserAvatar called") {
```

Comment:
I would arrange this tests in a different way: One test, instead of two, in which we check:
- if the storageReference.putBytes was called with specific value, this can be a mock, which is returned from bitmapConverter.convertBitmapToBytes when userPhoto and quality is passed there

Reply:
Done 😄